base/applications/notepad/main.c

```diff
@@ -656,5 +660,10 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE prev, LPTSTR cmdline, int sh
 
     DestroyAcceleratorTable(hAccel);
 
-    return (int) msg.wParam;
+#if defined(_MSC_VER) && defined(_DEBUG)
+    /* Report any memory leaks */
+    _CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
```

Comment:
By just setting it here you omit the error reporting on all failure cases (f.e. `HandleCommandLine` returning FALSE)

Reply:
Added `NOTEPAD_Main` and fixed the problem you mensioned.

Reply:
It would have been easier to just call that function at the start of winmain?

Reply:
OK.